### ISSUE TITLE
About the convergence curve of the training loss function

### ISSUE THREAD

**Issue by NNNNAI** [TencentARC/GFPGAN#92]
Hi xintao.
I have try to retrain the gfpganv1, but I noticed that the loss function about gan fluctuated significantly in the early stages of training as shown shown in the red box below.Could you give me a copy of the tensorboard file when you trained gfpganv1 so that I can compare the difference between the model I trained and yours？Thanks for your attention, have a nice day!~
![1635997159](https://user-images.githubusercontent.com/30747212/140254120-4b7d3893-7682-4cef-b318-6f12289d89b1.jpg)
![1635997159(1)](https://user-images.githubusercontent.com/30747212/140254129-087fac94-8d5a-4629-a6d5-3ad8b94692e4.jpg)
